glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt

```diff
@@ -798,8 +793,8 @@ class GleanTest {
         // real-world async behaviour of this.
         // We don't need to care about clearing it,
         // the test-unit hooks will call `resetGlean` anyway.
-        Dispatchers.API.setTaskQueueing(true)
-        Dispatchers.API.setTestingMode(false)
+        //Dispatchers.API.setTaskQueueing(true)
+        //Dispatchers.API.setTestingMode(false)
```

Comment:
Commented out and leaving it in?

Reply:
this can go.

Reply:
Ah! That's in the disabled test anyway. I left it in because if we ever re-enable that test it is good to know that we _used_ to require task qeueuing but not test mode.